cost-analyzer/templates/kubecost-admission-controller-template.yaml

```diff
@@ -7,11 +7,6 @@ metadata:
 webhooks:
   - name: "kubecost-deployment-validation.kubecost.svc"
     failurePolicy: Ignore
-    namespaceSelector:
-      matchExpressions:
-        - key: kubernetes.io/metadata.name
-          operator: In
-          values: [ "kubecost" ]
```

Comment:
Woah. I thought we were intentionally tightly scoping it to avoid security holes and then we were going to provide guidance on manual cert generation for users who want to increase the scope. What changed?

Reply:
Just general feedback/ confusion from people deploying to other namespaces and not seeing data. But I can revert and we can discuss later.